### DISCUSSION TITLE
Some icons are maked a deprecated.

### DISCUSSION THREAD

**Discussion by soufianeelc** [lucide-icons/lucide#1927] [Help] [answered]
I'm working with lucide-preact and I want to use some icons in my Astro.js project.

Let's take for exemple the `Instagram` icon, when I import it in my code `import { Instagram } from 'lucide-preact'` the IDE maks it as deprecated and I'm importing it the same way as mentionned in the [website](https://lucide.dev/icons/instagram)

what should I use instead?

![image](https://github.com/lucide-icons/lucide/assets/86732888/2dfd723b-3ee3-4509-bdf8-8155e6aa7339)

**Comment by karsa-mistmere** [comment, marked as answer]
Brand icons are deprecated and will be removed eventually, please refer to #670 
We suggest using simpleicons.org
